### PR TITLE
Fix test errors, and add ==null check.

### PR DESCRIPTION
**Pull request by Hzj-jie**
CI has been fixed, now we should have all tests pass.

### CODE REVIEW

File: spec/controllers/1/syncs_controller_spec.lua [hunk 1]
```diff
@@ -143,21 +143,28 @@ describe("SyncsController", function()
             update(username, userkey, doc, 0.32, "56", "my kpw")
             local response = get(username, userkey, doc)
             assert.are.same(200, response.status)
+            -- Clear timestamp, it varies.
+            response.body.timestamp = nil
```
Comment (houqp) [inline spec/controllers/1/syncs_controller_spec.lua]: At least check that timestamp is in the response?

Comment (Hzj-jie) [inline spec/controllers/1/syncs_controller_spec.lua]: Sorry, I cannot run travis case before pulling the request...

**Comment by houqp** [comment]
Are you not able to run the test case locally?

**Comment by Hzj-jie** [comment]
Not really, I am using a managed corp machine (do not tell my boss). So I cannot install some of the packages required.

**Comment by chrox** [comment]
Looks good now, merging.

**Comment by houqp** [comment]
LOL, use docker if it's a Linux machine ;)

**Pull request merged**